ev3dev/core.py

```diff
@@ -958,93 +962,84 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
-    def set_power(self, power):
-        self.speed_sp = power
-
-    def set_position_rotations(self, power, rotations):
+    def _set_position_rotations(self, power, rotations):
         if power > 0:
             self.position_sp = self.position + int(rotations * self.count_per_rot)
         else:
             self.position_sp = self.position - int(rotations * self.count_per_rot)
 
-    def set_position_degrees(self, power, degrees):
+    def _set_position_degrees(self, power, degrees):
```

Comment:
Can we have `rotations` call `degrees` instead? It makes more sense to me to do everything in terms of the "smaller" units and convert from the "larger" ones as needed. Or, it might be optimal to just make them operate independently so we avoid unnecessary nesting for execution speed.

Reply:
1 rotation, 360 degrees....rotation is the smaller unit :)

Reply:
If you need more degrees than rotations to represent the same quantity, the quantity represented by one degree is smaller than the quantity represented by one rotation.

Reply:
fixed